lib/trento.pm

```diff
@@ -268,16 +268,6 @@ sub cluster_config {
 
     $variables{HANA_ACCOUNT} = get_required_var("TRENTO_QESAPDEPLOY_HANA_ACCOUNT");
     $variables{HANA_CONTAINER} = get_required_var("TRENTO_QESAPDEPLOY_HANA_CONTAINER");
-    if (get_var("TRENTO_QESAPDEPLOY_HANA_KEYNAME")) {
```

Comment:
I'm not sure about what to do with Trento as I suppose is no more trivial to run a VR for this.

Reply:
yes, I planed to run VR for this, but failed like this http://openqaworker15.qa.suse.cz/tests/297639, it required trento_tools_.qcow2, but it seems that we don't provide it any more

Reply:
I'm not sure about it but I'd say if we cannot verify for the moment we should not change it.

Reply:
indeed, I revoked the code